Mathlib/Algebra/Polynomial/Module/AEval.lean

```diff
@@ -0,0 +1,197 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
```

Comment:
This is actually done by Richard in #7728.
```suggestion
Copyright (c) 2022 Richard M. Hill. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Richard M. Hill
```

Reply:
good catch!